### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by narekhovhannisyan**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

- Restructured application to include code in src folder.
- Updated `package.json` file.

### CODE REVIEW

**Comment by jawache** [comment]
I'll let @gnanakeethan review this one! 

@narekhovhannisyan can you link, refer this PR back to an issue, is this related to this one for instance? https://github.com/Green-Software-Foundation/ief/issues/77

**Comment by gnanakeethan** [comment]
@narekhovhannisyan 

I think you need to update the references to the build step in package.json for JSII.

File: src/lib/boavizta.ts [hunk 1]
```diff
@@ -1,7 +1,17 @@
-import {IImpactModelInterface} from './interfaces/index';
 import axios from 'axios';
 
-export {IImpactModelInterface} from './interfaces/index';
```
Comment (gnanakeethan) [inline src/lib/boavizta.ts:4]: @narekhovhannisyan This is essential for JSII to build.

**Review by gnanakeethan — CHANGES_REQUESTED** [review]

File: src/lib/boavizta.ts (hunk 1 above)
Comment (gnanakeethan) [reply src/lib/boavizta.ts:4]: [jsii.patch](https://github.com/Green-Software-Foundation/ief/files/12389418/jsii.patch)

Patch File for your reference. Few more changes are also aggregated here.

Comment (narekhovhannisyan) [reply src/lib/boavizta.ts:4]: Great, meanwhile checking build directory issue

**Comment by gnanakeethan** [comment]
@narekhovhannisyan I think we need to delete the index.ts in the src and move all the files inside the src/lib into src

**Comment by gnanakeethan** [comment]
[jsii.patch](https://github.com/Green-Software-Foundation/ief/files/12395485/jsii.patch)


Please check the above patch. We need this patched to do the "yarn package"

**Comment by narekhovhannisyan** [comment]
#77

**Review by gnanakeethan — APPROVED** [review]

**Comment by gnanakeethan** [comment]
Looks good to me. Compiled to JSII successfully.

**Pull request merged**